lib/ex338_web/views/user_view.ex

```diff
@@ -1,3 +1,11 @@
 defmodule Ex338Web.UserView do
   use Ex338Web, :view
+
+  alias Ex338.{User}
+
+  def user_profile_image(user = %User{}) do
```

Comment:
File has the variable name before the pattern while most of the files have the variable name after the pattern when naming parameter pattern matches